### DISCUSSION TITLE
Ingesting Crowdstrike endpoint data

### DISCUSSION THREAD

**Discussion by cyberkimh** [Security-Onion-Solutions/securityonion#4061] [Unsupported Versions]
Has anyone had much success ingesting Crowdstrike endpoint data into SO.2?

**Comment by weslambert** [comment]
We have not tried, but if you have any documentation from the Crowdstrike side, we could help to get you pointed in the right direction.

**Comment by Git-Me-Some-Hub** [comment]
@cyberkimh  - Did you see that Elastic has a module for Crowstrike? You can try and get it working on SO Sensor, you can have an external Filebeat server that forwards to Manager, or you can wait until 2.3.60 for Modules to be better integrated. 

https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-crowdstrike.html

**Reply by cyberkimh** [reply]
Thank you that's so helpful.
